src/entry-dev.js

```diff
@@ -11,7 +11,7 @@ import App from './App';
  */
 ReactDOM.render(
   <Provider store={ store }>
-    <Router basename='/insights/platform/approval'>
+    <Router basename='/insights/platform/rbac'>
```

Comment:
is basename required?

Reply:
I guess we will need something like this in Catalog

Reply:
I will have the change for /${RELEASE}/platform in a separate PR. This is already big as it is.